resources/qml/Validators/FloatValidator.qml

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2022 UltiMaker B.V.
```

Comment:
This tastes like a component which could/should live in UM

Reply:
I thought the same, but these validators specifically validate in the style we use for our settings. I think this is pretty Cura specific, since you can just use the QT validators otherwise.